src/lighthouse.php

```diff
@@ -449,4 +449,14 @@
          */
         'entities_resolver_namespace' => 'App\\GraphQL\\Entities',
     ],
+
+    /*
+    |--------------------------------------------------------------------------
+    | Print Schema Disk
+    |--------------------------------------------------------------------------
+    |
+    | Define what disk should be used when printing the schema to file.
+    |
+    */
+    'print_schema_disk' => env('LIGHTHOUSE_PRINT_SCHEMA_DISK', 'local'),
```

Comment:
Not a fan of this option, just the CLI flag should suffice.

Reply:
Sounds good